Tangem/Common/TangemSdk/CardInteractor.swift

```diff
@@ -11,13 +11,15 @@ import TangemSdk
 import Combine
 
 class CardInteractor {
-    private let tangemSdk: TangemSdk
-    private var cardId: String?
     private var cancellable: AnyCancellable?
+    private var input: Input
 
-    internal init(tangemSdk: TangemSdk, cardId: String?) {
-        self.tangemSdk = tangemSdk
-        self.cardId = cardId
+    internal init(cardInfo: CardInfo) {
+        input = .cardInfo(cardInfo)
+    }
+
+    internal init(cardId: String) {
+        input = .cardId(cardId)
```

Comment:
NP:  Я вообще думал что такое форматтер должен убрать, чекну при возможности

```suggestion
   init(cardInfo: CardInfo) {
        input = .cardInfo(cardInfo)
    }

   init(cardId: String) {
        input = .cardId(cardId)
```